.changeset/config.json

```diff
@@ -7,7 +7,6 @@
     }
   ],
   "ignore": [
-    "@contentful/f36-avatar",
     "@contentful/f36-image",
```

Comment:
We need to remove the image and the navbar from the ignore, to make the changeset config valid

Reply:
done